### DISCUSSION TITLE
Group Servers

### DISCUSSION THREAD

**Discussion by dev-jhon-yo** [stoatchat/discussions#417] [Feature Suggestions]
New Feature: 

Add a way to group the servers, being able to create categories/groups.  
In these groups/categories you can group servers according to the desired category.

**Comment by Pseudoexpertise** [comment]
Do you mean like the Discord Hubs?

**Reply by ghost** [reply]
I think he's talking about folders on the left server bar, to sort the servers (like on discord).

**Comment by LukarioMC** [comment]
+1, would like it if we can also at _least_ have one level of sub-folders. 

For example: on Discord, I am a part of way too many servers and they only have top-level folders which is very limiting in terms of grouping.
